plugins/pluginmanagers/lz-n.nix

```diff
@@ -11,6 +11,8 @@ nixvim.neovim-plugin.mkNeovimPlugin {
   name = "lz-n";
   originalName = "lz.n";
   maintainers = [ maintainers.psfloyd ];
+  # NOTE: We want to load lz.n as early as possible so that triggers are respected
+  configLocation = "extraConfigLuaPre";
```

Comment:
Makes me wonder if `configLocation` should support `mkOrder` wrappers?

To allow us to more explicitly say things like `configLocation = mkOrder 69 "extraConfigLuaPre"`?

Reply:
I think it would be useful